static/javascript/launch.js

```diff
@@ -543,6 +542,16 @@ function loadSupplementaryDataInfo() {
   }
 }
 
+function clearSDSMetadata() {
+  let dataset = document.querySelectorAll("#sds_data > div");
+  let div_array = [...dataset];
+  for (let i = 0; i < div_array.length; i++) {
+    if (div_array[i].classList.contains("sds-data")) {
```

Comment:
I think there should be a better way of working out if its the SDS data than a hardcoded class